src/usd/sdf_usd_parser/material.cc

```diff
@@ -0,0 +1,355 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include "sdf_usd_parser/material.hh"
+
+#include <map>
+
+#include <ignition/math/Color.hh>
+#include <pxr/base/tf/stringUtils.h>
+#include <pxr/usd/sdf/types.h>
+#include <pxr/usd/usd/prim.h>
+#include <pxr/usd/usdShade/connectableAPI.h>
+#include <pxr/usd/usdShade/materialBindingAPI.h>
+
+#include "sdf/Pbr.hh"
+
+namespace usd
+{
+  void RemoveProperty(
+    pxr::UsdStageRefPtr &_stage,
+    pxr::SdfPath prim_path,
+    pxr::TfToken property_name)
```

Comment:
Can you document this method?

Also, is it needed? It looks like the only place it was being called is currently commented out: https://github.com/ignitionrobotics/sdformat/blob/5e3f5fe96969b922934fa195c69fd38721620125/src/usd/sdf_usd_parser/material.cc#L76